src/main/java/org/sasanlabs/service/vulnerability/xss/reflected/XSSInImgTagAttributeInjectionVulnerability.java

```diff
@@ -42,11 +49,14 @@ public UrlParamBasedImgTagAttrInjection() {
     @VulnerableAppRequestMapping(value = LevelConstants.LEVEL_1, htmlTemplate = "LEVEL_1/XSS")
     public ResponseEntity<String> getVulnerablePayloadLevel1(
             @RequestParam Map<String, String> queryParams) {
+
```

Comment:
can you please solve this tech debt where we are taking a map of queryparams but actually we want to take the specific parameter which is `src`. 
something like `@RequestParam("src") String imagelocation`

Actual API call from Frontend: `http://localhost/VulnerableApp/XSSInImgTagAttribute/LEVEL_1?src=/VulnerableApp/images/ZAP.png`